Makefile

```diff
@@ -24,7 +24,7 @@ sort: Makefile
 		echo "  https://github.com/dealii/publication-list#installing-bibtool" >&2 ; \
 		exit 1 ; \
 	fi ; \
-	for f in publications-*.bib ; do \
+	for f in $(shell find . -type f -regex ".*\.bib") ; do \
 		sed -i 's/% Encoding: US-ASCII//' $$f ; \
 		bibtool -r bibtool.rsc -i $$f -o $$f || exit 1 ; \
```

Comment:
Instead of the contortion above, how about just doing
```
  bibtool -r bibtool.rsc -i ./$$f -o ./$$f || exit 1 ; \
```
here (i.e., adding `./`)?

Reply:
That's no problem. An extra ``./`` does not hurt.

I used this cumbersome expression, because in my case I have several subdirectories, where the ``*.bib`` files are located.